front/src/styles/scss/_overrideBootstrapSNCF.scss

```diff
@@ -69,8 +69,8 @@
 .form-control-container .chips-input {
   width: auto !important;
   flex-grow: 1;
-  min-height: 2.45rem;
   line-height: 1;
+  outline: none;
```

Comment:
Why ? We should never remove outline, it's needed for accessibility https://www.a11yproject.com/posts/never-remove-css-outlines/

Reply:
OK I removed this line, it was to remove the green border which is not very fancy, but it's ok :)